tools/owperf/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "owperf",
+  "version": "1.0.0",
+  "description": "",
+  "main": "owperf.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
fix license.

Reply:
see this file for example https://github.com/apache/incubator-openwhisk-composer/blob/master/package.json.